### ISSUE TITLE
The 3dvia game fix do not work in mitmproxy

### ISSUE THREAD

**Issue by livekn** [juvian/neopets-flash-fix#5]
The 3dvia game fix do not work in mitmproxy. It is because it's under `if "neopets.com" in flow.request.host:`, so never matched.

https://github.com/juvian/neopets-flash-fix/blob/bd3c8f2a5ecab922b0284a24de12aead9877e446/mitmproxy/neopets.py#L44

**Comment by juvian** [comment]
Good catch, [changed](https://github.com/juvian/neopets-flash-fix/commit/19e0212ecf78bcd95084995dcbf468746e4707b1) it but didn't try it
